TeapotTests/MockTests.swift

```diff
@@ -84,11 +84,34 @@ class MockTests: XCTestCase {
         mockedTeapot.get("/overridden") { (result: NetworkResult) in
             switch result {
             case .success(let json, _):
-                XCTAssertEqual(json!.dictionary!["overridden"] as! String, "value")
+                XCTAssertEqual(json?.dictionary?["overridden"] as? String, "value")
             case .failure(let error):
                 print(error)
                 XCTFail()
             }
         }
     }
+    
+    func testEndpointOverriddingThenHittingOtherEndpointWithError() {
```

Comment:
In these cases, I always find it very useful to document the test with the expected json I'm sending/receiving. Makes both following the code and checking it easier.